src/app/screens/Home/index.tsx

```diff
@@ -1,19 +1,23 @@
 import React, { useCallback, useRef, useEffect } from 'react';
 import { SafeAreaView, FlatList, ListRenderItem, View, ActivityIndicator } from 'react-native';
 import { useSelector, useDispatch } from 'react-redux';
+import { useNavigation } from '@react-navigation/native';
 import { State } from '@interfaces/reduxInterfaces';
 import { Article } from '@interfaces/articlesInterface';
 import { THRESHOLD } from '@constants/pagination';
```

Comment:
you didn't do this but can you sort the imports?

Reply:
Where should it go?